apps/condo/domains/meter/schema/MeterVerificationNotification.test.js

```diff
@@ -0,0 +1,115 @@
+const { prepareKeystoneExpressApp, setFakeClientMode } = require('@core/keystone/test.utils')
+const { createTestMeterWithResident } = require('../utils/testSchema')
+const { sendVerificationDateReminder } = require('@condo/domains/meter/tasks/sendVerificationDateReminder.js')
+const { Message: MessageApi } = require('@condo/domains/notification/utils/serverSchema/index.js')
+const { METER_VERIFICATION_DATE_REMINDER_TYPE } = require('@condo/domains/notification/constants/constants')
+const dayjs = require('dayjs')
+let keystone = null
+
+
+const getNotificationsFromMeter = async ({ verificationDate, nextVerificationDate, searchWindowDaysShift = 0 }) => {
+    const { resident: { user: { id } } } = await createTestMeterWithResident({ verificationDate, nextVerificationDate })
+    await sendVerificationDateReminder({ date: null, searchWindowDaysShift, daysCount: 30 })
+    return await MessageApi.getAll(keystone, { user: { id }, type: METER_VERIFICATION_DATE_REMINDER_TYPE })
+}
+
+describe('Meter verification notification', () => {
+
+    beforeAll(async () => {
+        setFakeClientMode(require.resolve('../../../index'))
+        const result = await prepareKeystoneExpressApp(require.resolve('../../../index'))
+        keystone = result.keystone
+    })
+
```

Comment:
It looks like a task.spec.js. Probably it's better to move it to the tasts.js file

Reply:
renamed